api/events/schema.py

```diff
@@ -74,6 +75,30 @@ def mutate(self, info, **kwargs):
         return CancelEvent(event=event)
 
 
+class EndEvent(graphene.Mutation):
+    """
+    Mutation to end an event
+    Returns event payload on ending the event
+    """
+    class Arguments:
+        calendar_id = graphene.String(required=True)
+        event_id = graphene.String(required=True)
+        start_time = graphene.String(required=True)
+        end_time = graphene.String(required=True)
+        meeting_end_time = graphene.String(required=True)
+    event = graphene.Field(Events)
+
+    def mutate(self, info, **kwargs):
+        room_id, event = check_event_in_db(self, info, "ended", **kwargs)
```

Comment:
Avoid using abbreviations like `db`

Reply:
@Bonifase This was an already existing method, should it be renamed?